examples/apis.json

```diff
@@ -0,0 +1,36 @@
+{
+    "functions": {
+        "base": "IsMoving",
+        "board": "GPIOPinByName",
+        "camera": "Properties",
+        "motor": "IsMoving",
+        "sensor": "Readings",
+        "servo": "Position",
+        "arm": "EndPosition",
+        "audio": "MediaProperties",
+        "gantry": "Lengths",
+        "gripper": "IsMoving",
+        "input_controller": "Controls",
+        "movement_sensor": "Properties",
+        "pose_tracker": "Poses",
+        "motion": "GetPose",
+        "vision": "DetectorNames"
+    },
+    "args": {
+        "base": "context.Background()",
+        "board": "",
+        "camera": "context.Background()",
+        "motor": "context.Background()",
+        "sensor": "context.Background(), map[string]interface{}{}",
+        "servo": "context.Background(), map[string]interface{}{}",
+        "arm": "context.Background(), map[string]interface{}{}",
+        "audio": "MediaProperties",
+        "gantry": "context.Background(), map[string]interface{}{}",
+        "gripper": "context.Background()",
+        "input_controller": "context.Background(), map[string]interface{}{}",
+        "movement_sensor": "context.Background(), map[string]interface{}{}",
+        "pose_tracker": "",
+        "motion": "",
```

Comment:
do these two not have args?

Reply:
both have arguments that require you to supply another component which is why we are leaving them empty and letting the user fill them in. The same is true for GPIOPinByName function from board